### PR TITLE
Move directory size exporter from kyma repository to the new own repository

### PR DESCRIPTION
**Pull request by hisarbalik**
**Description**

Changes proposed in this pull request:

- Source code move to the new repository
- Make targets are adapted to the new repository requirements
- Code owners updated

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/16852

### CODE REVIEW

File: milv.config.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#TODO: remove https://dashboard.kyma-project after it's available
```
Comment (skhalash) [inline milv.config.yaml:1]: I guess it's a leftover?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+.PHONY: fmt
+fmt: ## Run go fmt against code.
+	go fmt ./...
+
+.PHONY: vet
+vet: ## Run go vet against code.
+	go vet ./...
+
+.PHONY: tidy
+tidy: ## Check if there any dirty change for go mod tidy.
```
Comment (skhalash) [inline Makefile:47]: Please add `tidy` to every target that depends on `fmt vet` (`run`, `build`, etc.)

**Review by skhalash — CHANGES_REQUESTED** [review]
Let's add a Dependabot configuration

File: milv.config.yaml (hunk 1 above)
Comment (hisarbalik) [reply milv.config.yaml]: removed

File: Makefile (hunk 2 above)
Comment (hisarbalik) [reply Makefile:47]: tidy called added to targets `test`, `build` and `run`

**Comment by hisarbalik** [comment]
> Let's add a Dependabot configuration

dependabot configuration added

**Review by skhalash — APPROVED** [review]

**Pull request merged**